src/nmea.h

```diff
@@ -32,11 +32,23 @@
 #define NMEA_GGA_FIX_MANUAL  7
 #define NMEA_GGA_FIX_SIM     8
 
+#define RAD2DEG_INT(X) (s16)((180.0 / M_PI) * X)
+#define RAD2DEG_FLOAT(X) ((180.0/M_PI) * X)
+#define MINUTES(X) fabs(60 * (RAD2DEG_FLOAT(X) - RAD2DEG_INT(X)));
+#define MS2KNOTTS(x,y,z) sqrt(x*x + y*y + z*z) * 1.94385
+#define MS2KMHR(x,y,z) sqrt(x*x + y*y + z*z) * (3600/1000)
```

Comment:
These would eventually be candidates to move to libswiftnav. Also you should put parens around macro function arguments to prevent strangeness with operator precedence:

`#define MS2KNOTTS(x,y,z) sqrt((x)*(x) + (y)*(y) + (z)*(z)) * 1.94385`